components/navbar/Navbar.tsx

```diff
@@ -0,0 +1,142 @@
+import Link from 'next/link';
+import Typography from '@material-ui/core/Typography';
+import Grid from '@material-ui/core/Grid';
+import { makeStyles, createStyles, Theme } from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    root: {
+      padding: '1rem 1.5rem',
+    },
+    middleGridContainer: {
+      justifyContent: 'space-between',
+    },
+    rightGridContainer: {
+      justifyContent: 'center',
+    },
+    logo: {
+      paddingLeft: '1.5rem',
+      width: '120px',
+      height: '35px',
+    },
+    a: {
+      textDecoration: 'none',
+    },
+    link: {
+      color: theme.palette.text.primary,
+      padding: '0.5rem 1rem',
+      borderRadius: '20px',
+      fontSize: '1rem',
+      letterSpacing: '0.05rem',
+      transition: 'color 0.2s, background 0.2s',
+      '&:hover': {
+        color: 'rgb(255, 255, 255)',
+        background: theme.palette.primary.main,
+      },
+    },
+    linkBold: {
+      fontWeight: 'bold',
+    },
+  }),
+);
+
+const Navbar = (): React.ReactElement => {
+  const classes = useStyles();
+
+  return (
+    <Grid
+      container
+      direction="row"
+      justify="space-between"
+      alignItems="center"
+      className={classes.root}
+    >
+      <Grid item lg={2}>
+        <img
+          src="/kara_gradient.png"
+          alt="kara logo"
+          className={classes.logo}
+        />
+      </Grid>
+      <Grid
+        item
+        lg={6}
+        container
+        spacing={1}
+        className={classes.middleGridContainer}
+      >
+        <Grid item>
+          <Link href="/">
+            <a className={classes.a}>
+              <Typography variant="h6" className={classes.link}>
+                Home
+              </Typography>
+            </a>
+          </Link>
```

Comment:
Regarding all Link Components: 
- Forgot to say that after implementing the translation thing, we have to use a Link from "i18n" instead of next.js
- Alex created a Button component that can get an href. So it also returns a Link Component if you want.
- I changed our Button Component so that it uses the Link from i18n

--> We need to either import the Link from i18n and use it. Or maybe better: use our own Button component and pass it an href